docker/observability/otel/otel-collector-config.yml

```diff
@@ -35,7 +41,9 @@ exporters:
     endpoint: 0.0.0.0:8889
     namespace: otel  # this becomes a prefix in the prometheus metrics: `otel_*`
     const_labels:
-      environment: docker
+      environment: local_otel_collector
+    resource_to_telemetry_conversion:
```

Comment:
isn't it duplicating of what you do in "metric_statements"? Does it mean that the metric datapoints would have both "of.client" and "client" labels?

Reply:
these are fixed labels that are added to the metrics. they can help point back to the origin, because prometheus scrapes metrics from many sources:

* the node-exporter that runs on servers, for each server
* the container metrics (`cAdvisor`) for every container on every server
* itself
* the open telemetry collector running on the same host as prometheus (localhost:8889), which exposes the metrics from all open forms services/instances and in the future also other projects like OIP

Reply:
> isn't it duplicating of what you do in "metric_statements"? Does it mean that the metric datapoints would have both "of.client" and "client" labels?

misread the question - this `resource_to_telemetry_conversion` is necessary to get the labels in the first place! The prometheus config then needs some tweaking to drop irrelevant labels. In this sample set up it will indeed have both the `client` and `of_client` labels, and the same for `target`.